src/Microsoft.DotNet.Build.CloudTestTasks/CreateAzureContainer.cs

```diff
@@ -77,18 +82,49 @@ public override bool Execute()
 
         public async Task<bool> ExecuteAsync()
         {
+            if (!string.IsNullOrEmpty(ConnectionString))
+            {
+                if (!(string.IsNullOrEmpty(AccountKey) && string.IsNullOrEmpty(AccountName)))
+                {
+                    Log.LogError("If the ConnectionString property is set, you must not provide AccountKey / Name.  These values will be deprecated in the future.");
+                    return false;
```

Comment:
I know we do this everywhere, but the preferred method is to return `!Log.HasLoggedErrors`